docs/source/tutorials/configuring_noise.rst

```diff
@@ -0,0 +1,165 @@
+To do this, we will pass some **configuration** to the :func:`pseudopeople.generate_current_population_survey` function.
+
+.. code-block:: python
+
+    cps_2025 = psp.generate_current_population_survey(year=2025, config={
+        'current_population_survey': {
+            'column_noise': {
+                'last_name': {
+                    'use_fake_name': {
+                        'cell_probability': 0.3,
+                    },
+                },
+            },
+        },
+    })
+
+In English, our configuration says: in the **CPS** dataset, for the **last name** column, the **fake name** noise type's **cell probability**
+parameter should be 0.3 (30%).
+The full set of parameters for each noise type is documented at :ref:`Noise Type Details <noise_type_details>`.
+The :code:`column_noise` key specifies that we are configuring column-based noise; the categories of noise are explained in more detail
+on the :ref:`noise page <noise_main>`.
+
+Let's take a look at the names in our generated CPS dataset:
+
+.. code-block:: pycon
+
+    >>> cps_2025[['first_name', 'middle_initial', 'last_name']]
+       first_name middle_initial          last_name
+    0      Regina              S                Coh
+    1         Pat              J              Posey
+    2   Stephanie              Y             Martin
+    3      Alaina              S               Resp
+    4       Haley              T                Goh
+    5     Katrina              V            Deltoro
+    6     Gregory              D             Talley
+    7   Catherine              R           Benjamin
+    8      Dustin              J           Benjamin
+    9        John              S              Bakal
+    10  Catherine              K            Trexler
+    11     Tamela              V             Female
+    12       Reed              J                  F
+    13    William              D            Masters
+    14    Michael              D      Serna Marquez
+    15    Addison              I            Thomson
+    16     Denise              A     Harting-Conant
+    17      David              J                Goh
+    18       Mary              A  Lady Od The House
+    19     Pamela              M               Reed
+    20       Alex              S             Ojukwu
+
+As expected, we see a number of strings in the last name column that are unlikely to be true last names.
+We can check exactly which ones are fake names by comparing to the same dataset without fake name noise in
+the last name column.
+For brevity, we do not show the steps to do this here, but
+we would find that there are seven such strings, which is almost exactly 30% of our 21 respondents.
+
+Increasing noise in first names
+-------------------------------
+
+Imagine we also want to increase the probability of a fake first name from its default of 1%.
+We can do this by using a modified configuration dictionary:
```

Comment:
```suggestion
We can do this by modifying the configuration dictionary. This time, we'll save the configuration dictionary to a variable before using it to generate the dataset:
```
I thought an extra sentence might be helpful to avoid confusion about why the code looks different this time. Feel free to re-word if you have a better idea.